roles/docker-engine/tasks/Debian.yml

```diff
@@ -5,5 +5,15 @@
 - name: Add Docker APT repo
   apt_repository: repo="deb https://apt.dockerproject.org/repo {{ ansible_distribution | lower }}-{{ ansible_distribution_release }} main" state=present
 
+- name: check local cassandra instance
+  shell: curl -m 0.01 localhost:9042 -v -s 2>&1|grep Connect
+  register: cassandra_ping
+
 - name: Install docker
   apt: name=docker-engine force=yes state=latest
+  register: update_docker_flag
+
+- name: wait for cassandra if docker was re-deployed
```

Comment:
Ideally we'd drain the agent node before upgrading docker. Part of that would be to increase the timeout to gracefully stop dockerized tasks. Not sure how this can be done though.